doc/mpich/tuning_parameters.md

```diff
@@ -704,6 +704,23 @@ as shown in Table V.
 + ---------- + ----------------------------------------------- + ------------------------------------ +
 ```
 
+
+
+### 2.7. Fallback Behavior for Collective Algorithm
+ 
+MPICH will fallback if the selected algorithm is not applicable to the
```

Comment:
Can we improve this description of when fallback occurs? I think we should state that fallback can occur from user-specified selections **and** default selections from the .json configurations **and** CVAR overrides. I think it is slightly counter-intuitive that even if you force a particular algorithm with a CVAR, you still may fallback, and that should be clear here.